File: op-black-frontend/src/components/Card.ts
export default class Card {
  suit: string;
  number: string;

  constructor(suit: string, number: string) {
    this.suit = suit;
    this.number = number;
  }

  toString(): string {
    return `${this.number} of ${this.suit}`;
  }

  value(): number {
    if (["Jack", "Queen", "King"].includes(this.number)) {
      return 10;
    } else if (this.number === "Ace") {
      return 11; // or 1, this possibility will be calculated in the game class
    } else {
      return parseInt(this.number);
    }
  }
}

File: blackjack-sim-python/card.py
class Card:
    def __init__(self, suit, number):
        self.suit = suit
        self.number = number

    def __str__(self):
        return f"{self.number} of {self.suit}"

    def value(self):
        if self.number in ['Jack', 'Queen', 'King']:
            return 10
        elif self.number == 'Ace':
            return 11  # or 1, this possiblity will be calculated in game class
        else:
            return int(self.number)